faunadb/client.py

```diff
@@ -57,6 +87,7 @@ def __init__(
       observer=None,
       pool_connections=10,
       pool_maxsize=10,
+      last_txn_time=None,
```

Comment:
you're passing the value via argument but is not using it, also those shared states we are passing via `kwargs`

Reply:
`kargs` makes sense. It hides the "dirty state" from the API. Users should have to interact with the `last_txn_time`. Maybe we should make it private?

Reply:
by "make it private", do you mean prepend the identifier with an underscore?